src/mme-app/utils/mmeTimerUtils.cpp

```diff
@@ -64,12 +68,19 @@ uint32_t MmeTimerUtils::stopTimer(TimerContext* timerCtxt)
 
 void MmeTimerUtils::onTimeout(TimerContext* timerCtxt)
 {
+    log_msg(LOG_DEBUG, "\n %s : %d \n",__FUNCTION__,__LINE__);
     MmeUeTimerContext* mmeTimerCtxt = static_cast<MmeUeTimerContext *>(timerCtxt);
     if (mmeTimerCtxt == NULL)
     {
         return;
     }
+    if(mmeTimerCtxt->getTimerId() == mmeConfigDnsResolve_c)
+    {
+        log_msg(LOG_DEBUG,"DNS resolution timeout, Let's try one more time ");
+        mme_tables->initiate_spgw_resolution();
+    }
 
+#if 0
```

Comment:
eRAB modification changes had enabled this block.
Perhaps we should invoke separate API based on the timer type.